icicle/primitives/field.cuh

```diff
@@ -23,6 +23,23 @@ template <class CONFIG> class Field {
       return Field { CONFIG::one };
     }
 
+    static constexpr HOST_DEVICE_INLINE Field from_limbs(uint32_t * limbs) {
```

Comment:
I don't think I see this method used in the codebase.

Reply:
Removed in 24222badb6d0db93bc0f0645ef7103beb4a6fa30